src/templates/Angular InstantSearch/src/index.html

```diff
@@ -6,6 +6,8 @@
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://cdn.jsdelivr.net/npm/instantsearch.css@7/themes/reset.css" rel="stylesheet" />
```

Comment:
I think reset is not necessary if you add satellite, as it includes the rules for the former.

Reply:
Oh that's true! thanks! fixed in f309b40